docs/sidebars.js

```diff
@@ -146,7 +146,8 @@ const sidebars = {
         'widgets/timer',
         'widgets/toggle-switch',
         'widgets/tree-select',
-        'widgets/vertical-divider'
+        'widgets/vertical-divider',
+        'widgets/pagination'
```

Comment:
All the widgets and datasources in the sidebar are in lexicographic order.
Please place pagination accordingly.